B827EBFFFF33BF13.json

```diff
@@ -0,0 +1,18 @@
++{
+ +  "gateway_conf": {
```

Comment:
Please check the content of the file, you copy&pasted the plus signs of another changeset, making this an invalid JSON file :)

Reply:
Just noticed it myself. I will change the file now. Sorry, I'm new to github :)